cli/command/image/trust.go

```diff
@@ -28,8 +28,8 @@ type target struct {
 	size   int64
 }
 
-// trustedPush handles content trust pushing of an image
-func trustedPush(ctx context.Context, cli command.Cli, repoInfo *registry.RepositoryInfo, ref reference.Named, authConfig types.AuthConfig, requestPrivilege types.RequestPrivilegeFunc) error {
+// TrustedPush handles content trust pushing of an image
+func TrustedPush(ctx context.Context, cli command.Cli, repoInfo *registry.RepositoryInfo, ref reference.Named, authConfig types.AuthConfig, requestPrivilege types.RequestPrivilegeFunc) error {
```

Comment:
I think we certainly can for the `GetSignableRoles` function and so I've moved over.

This one is a little more tricky because it calls `imagePushPrivileged` - can we export that function?

Reply:
Ya, I think it makes sense to export it. There's nothing in there that's really specific to the command routing. The only change I would make is to accept a `client.ImageAPIClient` instead of the full `command.Cli`